mne/stats/regression.py

```diff
@@ -387,3 +389,27 @@ def _make_evokeds(coefs, conds, cond_length, tmin_s, tmax_s, info):
             kind='average')  # nave and kind are technically incorrect
         cumul += tmax_ - tmin_
     return evokeds
+
+
+def _get_solver(solver='cholesky'):
+    if callable(solver):
+        return(solver)
+    elif isinstance(solver, string_types):
```

Comment:
if someone passes a `solver` that is neither string nor callable, then this function will return `None`